db/migrate/20200103170411_add_allow_prompt_to_service_providers.rb

```diff
@@ -0,0 +1,10 @@
+class AddAllowPromptToServiceProviders < ActiveRecord::Migration[5.1]
+  def up
+    add_column :service_providers, :allow_prompt, :boolean
```

Comment:
How would you feel about renaming this `allow_prompt_login` or something like that since there are multiple values for the `prompt` param.

Reply:
Sure, good point!